src/components/collections-grid/collections-grid.module.scss

```diff
@@ -1,82 +1,86 @@
 @import '../../base/module';
 
 .ShopByRoom {
+  background-color: white;
   padding: 80px 0;
+  text-align: center;
   h2 {
     margin-bottom: 60px;
   }
   .collectionsGrid {
     display: grid;
     grid-template-columns: repeat(2, 1fr);
-    grid-template-rows: repeat(4, 1fr);
-    column-gap: 20px;
+    grid-template-rows: repeat(2, 1fr);
+    column-gap: 40px;
     row-gap: 30px;
+    padding: 0 20px;
+    margin-bottom: 50px;
 
     @media (min-width: $tablet) {
       grid-template-columns: repeat(4, 1fr);
-      grid-template-rows: repeat(2, 1fr);
+      grid-template-rows: repeat(1, 1fr);
+      column-gap: 70px;
+      margin: 70px 0px;
+      padding: 0 50px;
     }
-  
-    & > a:nth-child(1) {
-      grid-area: 1 / 1 / 3 / 3;
-      .imgBox {
-        aspect-ratio: 8/9;
-      }
-
-      @media (min-width: $tablet) {
-        grid-area: 1 / 1 / 3 / 3;
-      }
-    }
-    & > a:nth-child(2) {
-      grid-area: 3 / 1 / 4 / 2;
 
-      @media (min-width: $tablet) {
-        grid-area: 1 / 3 / 2 / 4;
-      }
-    }
-    & > a:nth-child(3) {
-      grid-area: 3 / 2 / 4 / 3;
-
-      @media (min-width: $tablet) {
-        grid-area: 1 / 4 / 2 / 5;
-      }
-    }
-    & > a:nth-child(4) {
-      grid-area: 4 / 2 / 5 / 3;
-
-      @media (min-width: $tablet) {
-        grid-area: 2 / 4 / 3 / 5;
-      }
-    }
-    & > a:nth-child(5) {
-      grid-area: 4 / 1 / 5 / 2;
-
-      @media (min-width: $tablet) {
-        grid-area: 2 / 3 / 3 / 4;
-      }
-    }
     .gridItem {
       .imgBox {
         width: 100%;
         aspect-ratio: 1/1;
         position: relative;
         overflow: hidden;
-        background: white;
+        background: $color-sagelt;
       }
       img {
         width: 100%;
         height: 100%;
+        border-radius: 250px;
```

Comment:
This totally works but to keep things fully rounded, feel free to just do like 99em here to keep things fully round no matter what the size.